### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by Watson1978**

### CODE REVIEW

File: output/copy.md [hunk 1]
```diff
@@ -111,7 +111,7 @@ This parameter is deprecated since v1.8.1. Use `copy_mode` instead.
 
 `out_copy` shares a record between `store` plugins by default.
 
-If `true`, `out_copy` passes dupped record to each `store` plugin. This behavior is similar to `copy_mode shallow`.
+If `true`, `out_copy` passes dumped record to each `store` plugin. This behavior is similar to `copy_mode shallow`.
```
Comment (kenhys) [inline output/copy.md]: duplicated? (or intended dup in Ruby context)

Comment (daipom) [inline output/copy.md]: Looks like this is not a typo.
I think `dupped` wants to say Ruby's `dup`.

Comment (Watson1978) [reply output/copy.md]: Ah. I see. Thanks

**Review by daipom — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**